### PR TITLE
GitHub Star Icon

### PR DESCRIPTION
**Pull request by Bgreen23**
This PR adds GitHub star icon to hub cards

Preview: https://5e8c84fcd0145b018028de46--shiftlab-pytorch-github-io.netlify.com/hub/research-models

### CODE REVIEW

**Review by patmellon — CHANGES_REQUESTED** [review]
Can you move the icon and star count closer to the card title, like in this example: https://5cba3680e9ee4802737a49d3--shiftlab-pytorch-github-io.netlify.com/pytorch-hub.

**Comment by Bgreen23** [comment]
Updated Preview: https://5e8ccc3ba422ea760f8d49fb--shiftlab-pytorch-github-io.netlify.com/hub/

**Review by patmellon — CHANGES_REQUESTED** [review]
When I hover over the star count number it turns orange. Can you make sure the font color stays gray?

File: _includes/compact_hub_cards.html [hunk 1]
```diff
@@ -8,7 +8,15 @@
...
-              <h4 class="compact-item-title col-6">{{ item.title }}</h4>
+              <div class="compact-hub-card-title-container">
+              <h4 class="compact-item-title">{{ item.title }}</h4>
+              <div class="detail-stars-container col-2">
+                <div class="github-stars-image">
+                  <img src="/assets/images/github-star.svg">
+                </div>
+                <div class="github-stars-count"></div>
+              </div>
+              </div>
```
Comment (patmellon) [inline _includes/compact_hub_cards.html]: nit - can you double-check the indentation here? It looks like the `detail-stars-container` div needs to be indented.

**Review by patmellon — CHANGES_REQUESTED** [review]

**Review by patmellon — APPROVED** [review]

**Pull request merged**